SUPPORT.md

```diff
@@ -0,0 +1,29 @@
+# Support
+
+## How to file issues and get help  
+
+Customers with a [support plan](https://azure.microsoft.com/en-us/support/options/) can open an Azure Support ticket [here](https://azure.microsoft.com/support/create-ticket/).
+**We recommend this option if your problem requires immediate attention.** 
+
+### Community resources
```

Comment:
Have we reviewed the order of the items in these resources? It feels weird that we ask folks to tweet the question before looking at this GitHub repo. Also, why is logging an issue in this repo not an option here?

Reply:
I'm open to change the order if you have suggestions. There is an option to search through issues and also GitHub issue is listed as a separate option below.

Reply:
My concern is that the note on GitHub issues should not be the last thing we mention in the support doc after community resources.

You can see my recommendation at https://github.com/Azure/azure-sdk-for-js/pull/15262